packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -593,7 +596,7 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
 
     protected renderResultLineNode(node: SearchInWorkspaceResultLineNode): React.ReactNode {
         const prefix = node.character > 26 ? '... ' : '';
-        return <div className={`resultLine noWrapInfo ${node.selected ? 'selected' : ''}`}>
+        return <div className={`resultLine noWrapInfo ${node.selected ? 'selected' : ''}`} title={`${node.lineText}`}>
```

Comment:
`node.lineText` already returns a `string` so no need to encapsulate it in quotations.
Also, due to formatting, I got better results when trimming the string so that it didn't have odd alignment within the title. With your code I was able to see `title` which had a lot of whitespace at the beginning, something we do not want to see in the end.

I changed this line to:

```ts
title={node.lineText.trim()}
```

Reply:
If you're curious, this is the problem with the original implementation, notice the spacing:

<div align='center'>

![a](https://user-images.githubusercontent.com/40359487/60591113-ba0d3500-9d6b-11e9-9221-a021be977ad9.gif)


</div>

Reply:
Thanks for your advice, I had fixed it :)